attribute/set.go

```diff
@@ -218,7 +227,9 @@ func NewSetWithFiltered(kvs []KeyValue, filter Filter) (Set, []KeyValue) {
 	if len(kvs) == 0 {
 		return empty(), nil
 	}
-	return NewSetWithSortableFiltered(kvs, new(Sortable), filter)
+	srt := sortables.Get().(*Sortable)
+	defer func() { sortables.Put(srt) }()
+	return NewSetWithSortableFiltered(kvs, srt, filter)
```

Comment:
I suggest doing the following 

```suggestion
	srt := sortables.Get().(*Sortable)
	s := NewSetWithSortableFiltered(kvs, srt, filter)
	sortables.Put(srt)
	return s
```

Reasons:

1. It would be the same pattern as in `func NewSet()`
2. A tiny performance improvement

Reply:
The performance overhead is ridiculously small (but there is) 😉 

https://www.sobyte.net/post/2022-01/go-defer-principle/#open-coding-1

Also from https://go.dev/doc/go1.14:

> This release improves the performance of **most** uses of defer to incur **almost** zero overhead compared to calling the deferred function directly.